pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -235,10 +235,68 @@ var logger *logrus.Logger
 var log *base.LogObject
 var zedcloudCtx *zedcloud.ZedCloudContext
 
+type infoDest uint
+
+// destination types, where info should be sent
+const (
+	ControllerDest  infoDest = 1
+	LocalServerDest          = 2
```

Comment:
Let's maybe start consistently denoting Local Profile Server with LPS (similalry to LOC abbreviation). Se here I would suggest `LPSDest` instead.

Reply:
Yep. Do we have shorter name for the "Controller"?

Reply:
No, we don't. But LPS is more accurate that LocalServer.